### PR TITLE
Pierogis!

### PR DESCRIPTION
**Pull request by Greniza**
## About The Pull Request

New imports from Space Kiev, pierogis! Now craftable.
pierogis1-big

## Why It's Good For The Game

mmm, pierogis...

## Changelog

:cl:
expansion: new food- pierogis!
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm [hunk 1]
```diff
@@ -237,3 +237,16 @@
 	)
 	result = /obj/item/food/bechamel_sauce
 	subcategory = CAT_MISCFOOD
+
+/datum/crafting_recipe/food/piesrogis
```
Comment (Amrabol) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm:241]: Typo

```suggestion
/datum/crafting_recipe/food/pierogis
```

Comment (Greniza) [reply code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm:241]: fuck

File: code/game/objects/items/food/misc.dm [hunk 2]
```diff
@@ -949,3 +949,11 @@
 	icon_state = "ready_donk_warm_mex"
 	tastes = list("enchiladas" = 2, "laziness" = 1)
 	foodtypes = GRAIN | DAIRY | MEAT | VEGETABLES | JUNKFOOD
+
+/obj/item/food/pierogis
+	name = "pierogis"
+	desc = "Potato and cheese dumplings, straight from Space Kiev. Served with caramelized onions and a healthy dallop of sour cream."
```
Comment (jjpark-kb) [inline code/game/objects/items/food/misc.dm:955]: This is like super super minute... but if this is somewhat in support of Ukraine (with the whole flag thing), I recommend changing Kiev to Kyiv.
```suggestion
	desc = "Potato and cheese dumplings, straight from Space Kyiv. Served with caramelized onions and a healthy dallop of sour cream."
```

Comment (Greniza) [reply code/game/objects/items/food/misc.dm:955]: I've seen it both ways; aren't they more or less interchangeable, and both acceptable?

Comment (jjpark-kb) [reply code/game/objects/items/food/misc.dm:955]: Kiev is the Russian way, Kyiv is the Ukrainian way.

**Comment by Mosslick** [comment]
Politics are cringe

**Review by Twaticus — CHANGES_REQUESTED** [review]
doesnt look like pierogis
no flag

**Comment by BlueMemesauce** [comment]
Pierogi is polish anyway

**Pull request closed without merging**